src/tools/kdb/recordundo.cpp

```diff
@@ -0,0 +1,47 @@
+/**
+ * @file
+ *
+ * @brief
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ */
+
+#include "recordundo.hpp"
+#include "coloredkdbio.hpp"
+#include <cmdline.hpp>
+#include <iostream>
+#include <kdb.hpp>
+#include <kdbrecord.h>
+
+using namespace std;
+using namespace kdb;
+
+RecordUndoCommand::RecordUndoCommand () = default;
+RecordUndoCommand::~RecordUndoCommand () = default;
+
+int RecordUndoCommand::execute (const Cmdline & cmdline)
+{
+	if (cmdline.arguments.size () > 1)
+	{
+		throw invalid_argument ("at max 1 argument needed");
```

Comment:
```suggestion
		throw invalid_argument ("max 1 argument needed");
```
To be consistent with the exception text in record-start.